config/superblock-order.ts

```diff
@@ -369,17 +369,16 @@ export const superBlockOrder: SuperBlockOrder = {
           SuperBlocks.DataAnalysisPy,
           SuperBlocks.InfoSec,
           SuperBlocks.MachineLearningPy,
-          SuperBlocks.CodingInterviewPrep
+          SuperBlocks.CodingInterviewPrep,
+          SuperBlocks.CollegeAlgebraPy,
```

Comment:
The test will pass if you swap these values.

```suggestion
          SuperBlocks.CollegeAlgebraPy,
          SuperBlocks.CodingInterviewPrep,
```